lib/miq_automation_engine/service_models/miq_ae_service_manageiq-providers-openshift-container_manager.rb

```diff
@@ -0,0 +1,12 @@
+module MiqAeMethodService
+  class MiqAeServiceManageIQ_Providers_Openshift_ContainerManager < MiqAeServiceManageIQ_Providers_ContainerManager
+    expose :container_image_registries, :association => true
+    expose :create_project
+    expose :delete_project
+    expose :projects_from_provider
+    expose :project_exists_in_provider?
+    expose :users_from_provider
+    expose :user_exists_in_provider?
+    expose :add_user_in_provider
```

Comment:
@moolitayer This code was originally in app/models/manageiq/providers/openshift/container_manager.rb.  However, the OpenShift provider was extracted a few days ago as a plugin.  Therefore, I fixed this up and added the code to that project.  Please see pull request https://github.com/ManageIQ/manageiq-providers-openshift/pull/2.